docs/structure-of-a-contract.rst

```diff
@@ -71,7 +71,7 @@ Function modifiers can be used to amend the semantics of functions in a declarat
     }
   }
 
- in the section on contracts for a more in-depth explanation.
+See into the section on :ref:`contracts` for a more in-depth explanation.
```

Comment:
It also has a reference. So, @michaltrzesimiech thanks for the PR, could you please change it to just delete that line?